diff --git a/app/routes/allocations.js b/app/routes/allocations.js
index 616f717..71419b6 100644
--- a/app/routes/allocations.js
+++ b/app/routes/allocations.js
@@ -9,13 +9,10 @@ function AllocationsHandler(db) {
     const allocationsDAO = new AllocationsDAO(db);
 
     this.displayAllocations = (req, res, next) => {
-        /*
+       
         // Fix for A4 Insecure DOR -  take user id from session instead of from URL param
         const { userId } = req.session;
-        */
-        const {
-            userId
-        } = req.params;
+        
         const {
             threshold
         } = req.query;
