diff --git a/server.js b/server.js
index d6bb500..ef42398 100644
--- a/server.js
+++ b/server.js
@@ -134,7 +134,7 @@ MongoClient.connect(db, (err, db) => {
     // Template system setup
     swig.setDefaults({
         // Autoescape disabled
-        autoescape: false
+        autoescape: true
         /*
         // Fix for A3 - XSS, enable auto escaping
         autoescape: true // default value
